sdk/test-utils/recorder-new/src/core-v1-recorder-node.ts

```diff
@@ -0,0 +1,86 @@
+// Copyright (c) Microsoft Corporation.
+// Licensed under the MIT license.
+
+import { URLBuilder } from "@azure/core-http";
+import { DefaultHttpClient, WebResource, WebResourceLike } from "@azure/core-http";
+
+const paths = {
+  playback: "/playback",
+  record: "/record",
+  start: "/start",
+  stop: "/stop"
+};
+export class RecordingHttpClient extends DefaultHttpClient {
+  private static s_recordingServerUri = "http://localhost:5000";
```

Comment:
I don’t particularly like the `s_` nomenclature here 🤔

Reply:
Also, what if this port is busy? what if users want to use some other host? shouldn’t these be parametrized?

Reply:
Still working on it. :)

Right now the recorder works for core-v1 and core-v2 for both node and browser.
For that, I had to write three different clients.

I'm on my way to merge all three into one.

Good point about parametrizing, will think about that.

Reply:
Added a point "Parametrize the host and port" in the original issue #15829